.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": true,
```

Comment:
This config differs from what's in github.com/entropic-dev/entropic. Would there be value in keeping them synced up?

Reply:
Makes sense.

Reply:
This is a great idea! I wasn't aware that project had prettier settings! Will update.